R/utilities-container.R

```diff
@@ -17,13 +18,20 @@
 #' containers <- getAllContainersMatching(c("Organism", "**", "Intracellular"), sim)
 #' @export
 getAllContainersMatching <- function(path, container) {
+  # Test for correct inputs
+  if (!isOfType(container, c("Simulation", "Container"))) {
+    stop(errorWrongType(container))
+  }
+  if (!is.character(path)) {
+    stop(errorWrongType(path))
+  }
+
   toContainers(rClr::clrCall(getContainerTask(), "AllContainersMatching", container$ref, path))
 }
 
 #' Retrieve a single container by path under the given container
 #'
-#' @param path Path of the container
-#' @param container The parent container used to find the container
+#' @inherit getAllContainersMatching

```

Comment:
We can inherit the documentation and just change the parts that are specific in this function.

Reply:
cool